Minimum in Rotated Sorted Array

```diff
@@ -0,0 +1,22 @@
+# time complexity o(log(n))
```

Comment:
Please explain what 'n' is

Reply:
'n' is the size of the given input, for suppose the given input has 'n' number of elements in it, size of the input is said to be 'n'.